### DISCUSSION TITLE
Explanation about LoRA merging into checkpoint

### DISCUSSION THREAD

**Discussion by Bullseye110** [hako-mikan/sd-webui-supermerger#356] [Q&A]
Hello!
I tried the "Merge to checkpoint" option from this extension and I used weights of 0.5 for 3 LoRAs, but then why I use the LoRAs keywords in the newly resulted base checkpoint, I do not get the style of the LoRAs at all.
Where is the problem and how should this be done
Thank you and I appreciate your help!

![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/137621616/387abe6b-6e3f-4735-b30b-d42b2261a90e)

**Comment by Dwanvea** [comment]
I'm wondering about this as well. Merging loras doesn't work at all. I tried everything and I still get completely different results for whatever reason.

**Comment by aleph23** [comment]
I have had luck merging lora to lora.  Best when they are very dissimilar.  When there is slight similarity it just makes for a mess.  There are exceptions, as long as all but one of the lora are fairly weak or overly generalized.  I use between 0.1 to full 1.0 for each lora weight, depending on it's strength and how much of it I want reflected in the final product.  Lots of trial and error.

However, merging lora into checkpoints I have had very little luck with and would also lover some advise!
